ToCgen.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash/env usr
+
+if test -f "ToC.md"; then
+    echo "ToC.md exist" |  rm ToC.md
+    echo "Updating new ToC.md" | touch ToC.md 
+else 
+    echo "Creating ToC.md" | touch ToC.md
+fi
+
+echo "## Overview
+Welcome to the DCP Vignette repository, containing walkthrough tutorials to help you get started with the DCP primarily via command-line access. For downstream application development, please refer to the [HCA DCP API documentation](https://prod.data.humancellatlas.org/apis)." >> ToC.md
+
+echo "The DCP welcomes any contributed notebooks or other tutorials to the list below. You can create your own branch and submit a pull request. " >> ToC.md
+
+echo "Vignettes" >> Toc.md
+
+find . -name "*.ipynb" | while read line; do
```

Comment:
I tested on my local machine, and the script is picking up some directories that it shouldn't. There isn't a compelling need to search recursively the entire repository hierarchy - we can replace this `find` call by doing

    g ls-files */{README.md,*.ipynb}

to find only notebooks or README.md files in the index.